bbb-export-annotations/workers/process.js

```diff
@@ -904,7 +904,8 @@ async function process_presentation_annotations() {
   // Launch Notifier Worker depending on job type
   logger.info(`Saved PDF at ${outputDir}/${jobId}/${filename_with_extension}`);
 
-  const notifier = new WorkerStarter({jobType: exportJob.jobType, jobId, filename: filename_with_extension});
+  const notifier = new WorkerStarter({jobType: exportJob.jobType, jobId,
+    fileNameToPath: filename_with_extension, filename: exportJob.filename});
```

Comment:
I suggest keeping the name `fileNameWithExtension` instead of `fileNameToPath`.

Reply:
Hum, but you mean to change just in the context of this file, right? Because otherwise, it wouldn't really make sense in the akka, for instance, since that `fileNameToPath` is the name that we want the file to have when we build the link to send it to the front-end.

Reply:
I would say to use this name in all contexts.
The name `fileNameToPath` does not allow me to predict exactly what information this property contains, whereas the name `fileNameWithExtension` is self-explanatory and clearly indicates the information contained in this property. It facilitates future maintenance.

Reply:
If I’m understanding correctly, fileNameToPath is the server-side filename used for the export - such as “annotated_slides”, as we cannot use user input there for security reasons. If that’s the case, I agree that we should rename it to something like serverSideFilename,  as it otherwise really isn’t clear.

Reply:
It is pretty much what Daniel said, I don't think `fileNameWithExtension` would be a good name mainly because it doesn't really carry the extension throughout the whole exporting flow, see https://github.com/bigbluebutton/bigbluebutton/pull/19541/files#diff-b1b4f70bcc656876c27640f28b099095132e2586258b1810ab7c93f4c3549b57R163 for example. As Daniel well explained, the name will be used only in the exporting flow. That being the case, I think `serverSideFilename` would be good, and indeed self-explanatory. What do you think @gustavotrott?

Reply:
Sure, it's better!